python/bruninga/packet.py

```diff
@@ -87,7 +87,9 @@ def to_bytes(self):
 
         # TODO: Handle various build conditions
         array += bytearray([self.control, self.protocol_id])
-        array += bytearray(self.info)
+        #print("packet info field:", self.info)
+        #array += bytearray(self.info, encoding='utf8')
```

Comment:
It's probably worth removing these commented-out lines (and the others in str_to_aprs.py).